docs/sources/operators-guide/architecture/components/query-frontend/index.md

```diff
@@ -82,10 +82,6 @@ To balance the number of workers connected to each query-frontend, the querier w
 If you run more query-frontend replicas than the number of workers per querier, the querier increases the number of internal workers to match the query-frontend replicas.
 This ensures that all query-frontends have some of the workers connected.
```

Comment:
```suggestion
This ensures that all query-frontends have some of the workers connected, but introduces a scalability limit because the more query-frontend replicas you run, the higher is the number of workers running for each querier, regardless of the configured `-querier.max-concurrent`.
```